suite-native/module-send/src/components/AddressOriginHelpButton.tsx

```diff
@@ -7,30 +7,48 @@ import { AddressReviewSheetSection } from './AddressReviewSheetSection';
 export const AddressOriginHelpButton = () => {
     return (
         <AddressReviewHelpSheet
-            title={<Translation id="moduleSend.review.address.originBottomSheet.title" />}
+            title={
+                <Text variant="titleSmall">
+                    <Translation id="moduleSend.review.address.originBottomSheet.title" />
+                </Text>
+            }
             subtitle={
-                <Text variant="body">
+                <Text>
                     <Translation id="moduleSend.review.address.originBottomSheet.subtitle" />
                 </Text>
             }
-        >
-            <AddressReviewSheetSection
-                title={
-                    <Translation id="moduleSend.review.address.originBottomSheet.exchange.header" />
-                }
-                content={
-                    <Translation id="moduleSend.review.address.originBottomSheet.exchange.body" />
-                }
-            />
+            body={
+                <>
+                    <AddressReviewSheetSection
+                        title={
+                            <Translation id="moduleSend.review.address.originBottomSheet.exchange.header" />
+                        }
+                        content={
+                            <Translation id="moduleSend.review.address.originBottomSheet.exchange.body" />
+                        }
+                    />
 
-            <AddressReviewSheetSection
-                title={
-                    <Translation id="moduleSend.review.address.originBottomSheet.person.header" />
-                }
-                content={
-                    <Translation id="moduleSend.review.address.originBottomSheet.person.body" />
-                }
-            />
-        </AddressReviewHelpSheet>
+                    <AddressReviewSheetSection
+                        title={
+                            <Translation id="moduleSend.review.address.originBottomSheet.person.header" />
+                        }
+                        content={
+                            <Translation id="moduleSend.review.address.originBottomSheet.person.body" />
+                        }
+                    />
+                </>
+            }
+        ></AddressReviewHelpSheet>
     );
 };
+
+// title={
+//     <VStack>
+//         <Text variant="titleSmall">
+//             <Translation id="moduleSend.review.address.originBottomSheet.title" />
+//         </Text>
```

Comment:
please, if there is no need to have this commented, remove it

Reply:
Completely obsolete. Removed in d6fb6a99c8.